### ISSUE TITLE
Replace fontawesome icons with USWDS

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#82075]
### User story

As a Cartography team member, I want to transition icons on the landing page and secondary nav to the new design system icons, so that the icons will stay the same after launch.

### Notes
- [clarifying thread](https://dsva.slack.com/archives/C0NGDDXME/p1714582450407219?thread_ts=1714572472.637479&cid=C0NGDDXME) with design system team (Matt)
- [Cartographers thread](https://dsva.slack.com/archives/C0NGDDXME/p1714582450407219?thread_ts=1714572472.637479&cid=C0NGDDXME) on options

Possible tasks:
- 

## Acceptance criteria

- Landing page renders the new USWDS icons
- Secondary nav renders the new USWDS icons
- QA review
- A11y review

**Comment by sterkenburgsara** [comment]
Update 5/14:
* Got sign-off on removing the mapped `medications` pill bottle icon from the new va-icon library and requesting to bring in a new icon from the Google material design icon library called `pill` for usage across VA.gov for prescriptions/medications, including within the secondary navigation component. [VADS ticket for that request is here.](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2811).  
* Also had a conversation in Slack about a discrepancy in the new library, whereby two icons are listed that both refer to appointments. We discussed this with Appointments and Auth experience teams, and OCTO design lead Kristen McConnell has a preference for the `calendar_today` icon between the two. [Filed another VADS ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2816) to request removal of the other `event_available` icon that should not be used for appointments at this time.

**Comment by sterkenburgsara** [comment]
We now have our four icons decided for secondary nav, and developers can move forward with updating those. Those icons are as follows (all should be used at the smallest icon size of 24 px): 
* Appointments: `calendar_today `
* Messages: `forum `
* Medications: `pill `(_still waiting on approval of this icon from VADS per ticket linked in comment above_)
* Medical records: `note_add`

We still need to work out the two remaining icons that will go on the My HealtheVet landing page for **Payments** & **Medical supplies** sections. 

cc @carlosfelix2 @jonathanjnelson

**Comment by sterkenburgsara** [comment]
Some options we've explored for icons under payments + medical supplies in Figma [here](https://www.figma.com/design/CAChU51fWYMZsgDR5RXeSc/MHV-Landing-Page?node-id=2245-20925&t=fCGT3JAkbuheczvF-4)

Will need to connect with auth experience team (and potentially other teams that may be affected if we want to consider alternative icons) to get their PoV.

**Comment by wesrowe** [comment]
I split the in-page icon updates into a [new ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83415)

**Comment by sterkenburgsara** [comment]
[Update on ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2811) about pulling in `pill` icon. Sounds like the DST will prioritize getting this into repos in their next sprint, which starts later this week. Wes is getting updates from DST team PM Carol Wong.

**Comment by wesrowe** [comment]
Hoping for an update tomorrow morning.

Note: we want to have the pill merged by Friday's go/no-go one way or another. So please move ahead with the work-around if it needs to be started to meet that timeline.

**Comment by radavis** [comment]
DST is on it: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2875

**Comment by radavis** [comment]
Update to component-library was released (v42.1.1). Unblocked 

https://github.com/department-of-veterans-affairs/component-library/releases/tag/v42.1.1

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that the pill icon for Medications is appearing as expected. All the other icons in the secondary nav are looking as expected as well

<img width="1525" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOU0yQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0ee1aee3ba82ce5c3f527b9c10323b66b676e535/Screenshot%202024-06-04%20at%2011.51.48%E2%80%AFAM.png" alt="Screenshot 2024-06-04 at 11.51.48 AM.png" />
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOVEyQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3542b3f00dce837663589c37a0fb914e19999113/Screenshot%202024-06-04%20at%2011.51.56%E2%80%AFAM.png" alt="Screenshot 2024-06-04 at 11.51.56 AM.png" />

**Comment by brea11y** [comment]
Verified that all icons (including the pill icon) have been swapped out with the new USWDS icons. Colors all match what is seen in Figma. No accessibility concerns.
